src/main/resources/screens/DCU_OFFLINE_QA_DETAILS.json

```diff
@@ -8,7 +8,7 @@
         "required"
       ],
       "props": {
-        "choices": "USERS_FOR_CASE"
+        "choices": "USERS_FOR_STAGE"
```

Comment:
It would be important for it to be noted that this change will have to go after the hocs-frontend/ hocs-casework ones due to the functionality required.

Reply:
Maybe add it on the ticket?

Reply:
Added as a comment on the ticket